source/atomics.tex

```diff
@@ -1148,6 +1148,7 @@
 \pnum
 \returns Atomically, the value pointed to by \tcode{object} or by \tcode{this} immediately before the effects.
 
+\indextext{signed integer representation!two's complement}%
```

Comment:
Actually, I think the index entries are better placed *after* the paragraph:

    \pnum
    \indextext{...}%
    Real text starts here